src/runtime/StaticStrings.h

```diff
@@ -765,6 +766,7 @@ namespace Escargot {
     F(PercentSign, "percentSign")                           \
     F(PlusSign, "plusSign")                                 \
     F(Quarter, "quarter")                                   \
+    F(Reject, "reject")                                     \
```

Comment:
`reject` string is already defined, so you can use it
https://github.com/Samsung/escargot/blob/1763b6f38da35c2b1c9eb8404ba84c41708b99e6/src/runtime/StaticStrings.h#L365